.gitignore

```diff
@@ -0,0 +1,22 @@
+# dependencies
```

Comment:
can you remove not important files? i think we only need to ignore few directories (node modules, dist)